CHANGELOG.md

```diff
@@ -167,6 +167,7 @@ See the [release announcement](https://godotengine.org/article/godot-4-1-is-here
 - Replace invalid characters in node names by underscores instead of removing them ([GH-75760](https://github.com/godotengine/godot/pull/75760)).
 - Optimize `Object::get_class_name` ([GH-75797](https://github.com/godotengine/godot/pull/75797)).
 - Ensure `RID`, `Callable`, and `Signal` are stored as strings ([GH-78517](https://github.com/godotengine/godot/pull/78517)).
+- The strings returned by `ResourceLoader::get_dependencies()` now include paths in addition to UIDs ([GH-73131](https://github.com/godotengine/godot/pull/73131)).
```

Comment:
The description looks good. This should be moved as first in the list, we order them by PR number.